lib/filterable_by.rb

```diff
@@ -33,7 +33,11 @@ def filterable_by(*names, &block)
 
       # @param [Hash] hash the filter params
       # @return [ActiveRecord::Relation] the scoped relation
-      def filter_by(hash)
+      def filter_by(hash = nil, **opts)
+        if hash.nil?
+          hash = opts
+          opts = {}
+        end
```

Comment:
I'm afraid this trick will fail for `xxx.filter_by(author_id: XXX)` - IIRC, `_filterable_by_config` holds field names as strings?

Reply:
request `params` have string keys (although with indifferent access). I didn't want to add an extra step of "stringifying" keys on input